drivers/input/keyboard_upper.c

```diff
@@ -60,6 +60,7 @@ struct keyboard_upperhalf_s
   struct list_node head;    /* Head of list */
   FAR struct keyboard_lowerhalf_s
                   *lower;   /* A pointer of lower half instance */
+  uint32_t         nums;    /* Number of buffer */
```

Comment:
```suggestion
  size_t           nums;    /* Number of buffer */
```
Or maybe we can reduce it to `uint16_t`?

Reply:
Ok, I change to uint8_t